ops/framework.py

```diff
@@ -613,7 +612,7 @@ def __init__(self, storage: Union[SQLiteStorage, JujuStorage],
 
         # Parse the env var once, which may be used multiple times later
         debug_at = os.environ.get('JUJU_DEBUG_AT')
-        self._juju_debug_at = (set(x.strip() for x in debug_at.split(','))
+        self._juju_debug_at = ({x.strip() for x in debug_at.split(',')}
```

Comment:
I do wonder if this is actually better syntax :) But it exists, and I wouldn't block if someone was using it.

Reply:
Yeah, same. But speaking of better syntax, looking at it again I didn't like the nested inline if-else with set comprehension, so I've also split it onto multiple lines. Hopefully clearer now.